verification/certifyKey.go

```diff
@@ -474,44 +488,48 @@ func checkCertificateStructure(t *testing.T, certBytes []byte) *x509.Certificate
 	return x509Cert
 }
 
-func testCertifyKey(d TestDPEInstance, client DPEClient, t *testing.T, simulation bool) {
-	ctx := getInitialContextHandle(d, client, t, simulation)
+func testCertifyKey(d TestDPEInstance, c DPEClient, t *testing.T, simulation bool) {
+	handle := getInitialContextHandle(d, c, t, simulation)
 	defer func() {
 		if simulation {
-			client.DestroyContext(ctx, DestroyDescendants)
+			c.DestroyContext(handle, DestroyDescendants)
 		}
 	}()
 
-	type Params struct {
-		Label []byte
-		Flags CertifyKeyFlags
-	}
-
 	profile, err := GetTransportProfile(d)
 	if err != nil {
 		t.Fatalf("Could not get profile: %v", err)
 	}
 	digestLen := profile.GetDigestSize()
 
+	var hashAlg asn1.ObjectIdentifier
+	if digestLen == 32 {
+		hashAlg = OidSHA256
+	} else if digestLen == 48 {
+		hashAlg = OidSHA384
+	} else {
+		t.Fatal("Unknown Hash Algorithm")
+	}
+
```

Comment:
This is required to build pub key from CertifyKey response which in turn is validated